hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenUpdateHandlerTest.java

```diff
@@ -267,6 +267,13 @@ void failsIfTokenImmutable() {
                 .get(fungibleTokenId)
                 .copyBuilder()
                 .adminKey((Key) null)
+                .wipeKey((Key) null)
```

Comment:
Should add unit tests to cover scenarios other than `failsIfTokenImmutable`

Reply:
Added